### DISCUSSION TITLE
With Gradle, tasks.quarkusBuild.nativeArgs seems ignored

### DISCUSSION THREAD

**Discussion by antoniomacri** [quarkusio/quarkus#28518] [Q&A]
In my Quarkus app, built with Gradle (and Kotlin), in the native artifact I need to include a resource from a dependency.

Specifically, `com.github.ua-parser:uap-java` contains a 
```
  private static final String REGEX_YAML_PATH = "/ua_parser/regexes.yaml";
```
in its jar. I try to specify the inclusion in the native build with any of these:
```
tasks.quarkusBuild {
    nativeArgs {
        "additional-build-args" to "-H:IncludeResources='.*/regexes.yaml'"
//        "resources.includes" to "**/ua_parser/regexes.yaml"
//        "resources.includes" to ".*/ua_parser/regexes.yaml"
//        "resources.includes" to "regexes.yaml"
    }
}
```
but none works. I run the native tests with:
```
./gradlew clean
./gradlew -i testNative
```

Instead, if I run with:
```
./gradlew clean
./gradlew -i -Dquarkus.native.additional-build-args=-H:IncludeResources='.*/regexes.yaml' testNative
```
it works since the build is done as expected:
```
docker run --env LANG=C --rm --user 1000:1000 ... -H:IncludeResources=.*/regexes.yaml ...
```

I'd like to specify the resource in the gradle config, not on the command line.

Am I missing something?

Reproducer: https://github.com/antoniomacri/quarkus-reproducer-native-resources

[a comment by an automated account is not shown]

**Comment by alex-cpm** [comment]
I use this expression:

quarkusBuild {
    nativeArgs {
        it.put('resources.includes', 'redisson.yaml,redisson-sentinel.yaml')
        it.put('container-build', 'true')
    }
}

**Reply by demtnt** [reply]
Could not convert it to kotlin-dsl style :(
